### PR TITLE
added support for mod_feedback_response_submitted

### PR DESCRIPTION
**Pull request by VassAngels**
hey @davidpesce, @ryansmith94 said to tag you as a reviewer here.

### CODE REVIEW

**Comment by davidpesce** [comment]
@VassAngels thanks for submitting this! I'm hoping I can get it reviewed this week. If not, I'll be at LSCon next week and it may have to wait until the following.

**Comment by davidpesce** [comment]
@VassAngels one other quick item. Can you take a look at the test that Ryan wrote? This PR will need the test associated with the event before it can be merged.

File: src/transformer/utils/constants.php [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 namespace transformer\utils;
 
-const xapi_type = 'http://id.tincanapi.com/activitytype/lms';
+const xapi_type = 'http://id.tincanapi.com/activitytype/lms/';
```
Comment (garemoko) [inline src/transformer/utils/constants.php:5]: why is this added?

Comment (ryasmi) [reply src/transformer/utils/constants.php:5]: Just temporary until we create a different URL for each type.

Comment (garemoko) [reply src/transformer/utils/constants.php:5]: "there's nothing as permanent as a temporary solution"

**Comment by ryasmi** [comment]
Copy, pasted, and tweaked this to work in the version 3 PR so closing this PR.
